Library/Formula/gitup.rb

```diff
@@ -0,0 +1,85 @@
+class Gitup < Formula
+  homepage "https://github.com/earwig/git-repo-updater"
+  url "https://github.com/earwig/git-repo-updater.git",
+    :revision => "51a482d7a25bed39ccc75780813c65bd666c1436",
+    :tag => "v0.2.3"
```

Comment:
You could use only of them here. The `tag` points to the revision so you can use that directly.